packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import Validate from '../../../Global';
+import InvalidClient from '../invalid_client';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import { parseToNewAPI, featureSwitchesURL } from '../../../Global';
+import FeatureRow from './row_switch';
+import { OverlayTrigger, Popover, Form, Alert } from 'react-bootstrap';
+import { FsContext } from '../../../feature_switches';
+import { useContext } from 'react';
+
+const popover = (onUpdate, errorMessage) => {
+    return (
+        <Popover id="popover-basic">
+            <Popover.Header as="h3">Want to update or remove?</Popover.Header>
+            <Popover.Body>
+                {errorMessage !== undefined ? <Alert variant="warning">{errorMessage}</Alert> : null}
+                <UpdateSwitch onUpdate={onUpdate} />
+            </Popover.Body>
+        </Popover>
+    );
+};
+const UpdateSwitch = ({ onUpdate }) => {
+    const [newSwitch, setNewSwitch] = useState({
+        is_enabled: 'true',
```

Comment:
Why are you using true as a string instead of a boolean?

Reply:
This is to set the default value for the dropdown menu

Reply:
This doesn't really answer my question. Why are you using string instead of a boolean value? Furthermore, as I think I mentioned somewhere below, this value should be derived from the state object and not be hardcoded.